etl/steps/data/grapher/ihme_gbd/2019/gbd_cause.meta.yml

```diff
@@ -5,39 +5,19 @@ definitions:
       topic_tags:
         - Diarrheal Diseases
         - Causes of Death
-
-  gbd_origin: &gbd_origin
-    producer: IHME, Global Burden of Disease Study (2019)
-    title: Institute for Health Metrics and Evaluation - Global Burden of Disease (2019) - Deaths and DALYs
-    description: >-
-      The Global Burden of Disease (GBD) provides a comprehensive picture of mortality and disability across countries,
-      time, age, and sex. It quantifies health loss from hundreds of diseases, injuries, and risk factors, so that health
-      systems can be improved and disparities eliminated.
-
-      GBD research incorporates both the prevalence of a given disease or risk factor and the relative harm it causes.
-      With these tools, decision-makers can compare different health issues and their effects.
-    citation_full: "Global Burden of Disease Collaborative Network. Global Burden of Disease Study 2019 (GBD 2019). Seattle, United States: Institute for Health Metrics and Evaluation (IHME), 2020."
-    url_main: https://www.healthdata.org/research-analysis/about-gbd
-    date_published: 2020-10-17
-    date_accessed: 2021-12-01
-    license:
-      url: https://www.healthdata.org/data-tools-practices/data-practices/terms-and-conditions
 dataset:
   update_period_days: 1095
 tables:
   diarrheal_diseases__both_sexes__age_standardized:
     variables:
-      ? deaths_that_are_from_diarrheal_diseases_per_100_000_people__in_both_sexes_aged_age_standardized
-      : sources: []
-        origins:
-          - *gbd_origin
+      deaths_that_are_from_diarrheal_diseases_per_100_000_people__in_both_sexes_aged_age_standardized:
+        sources: []
```

Comment:
Given that you already removed the sources with your `add_origins_to_global_burden_of_disease` function, this may not be necessary.